examples/stress_tests/many_buttons.rs

```diff
@@ -84,23 +89,24 @@ fn spawn_button(
     j: usize,
 ) {
     let width = 90.0 / total;
-    commands
-        .spawn((
-            ButtonBundle {
-                style: Style {
-                    size: Size::new(Val::Percent(width), Val::Percent(width)),
-                    bottom: Val::Percent(100.0 / total * i as f32),
-                    left: Val::Percent(100.0 / total * j as f32),
-                    align_items: AlignItems::Center,
-                    position_type: PositionType::Absolute,
-                    ..default()
-                },
-                background_color: color,
+    let mut builder = commands.spawn((
+        ButtonBundle {
+            style: Style {
+                size: Size::new(Val::Percent(width), Val::Percent(width)),
+                bottom: Val::Percent(100.0 / total * i as f32),
+                left: Val::Percent(100.0 / total * j as f32),
+                align_items: AlignItems::Center,
+                position_type: PositionType::Absolute,
                 ..default()
             },
-            IdleColor(color),
-        ))
-        .with_children(|commands| {
+            background_color: color,
+            ..default()
+        },
+        IdleColor(color),
+    ));
+
+    if std::env::args().nth(1).as_deref() != Some("no-text") {
```

Comment:
This extracts the flag for every button, Could we move this outside of this function and the spawning loop?